gateway/src/apicast/policy/caching/schema.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Caching policy configuration",
+  "type": "object",
+  "properties": {
+    "exit": {
+      "type": "caching_type",
+      "enum": ["resilient", "strict"]
```

Comment:
We could offer `none` as suggested in https://issues.jboss.org/browse/THREESCALE-587.

Reply:
:+1: